server/models/Identifier.js

```diff
@@ -27,25 +27,25 @@ const MealTimes = [
     }
 ];
 
-const Dates = [
+const datesLog = [
     {
-        value: moment(),
+        value: new Date().toISOString().split("T")[0],
         names: [
             "today"
         ]
     },
     {
-        value: moment().add(-1, 'day').toDate(),
+        value: moment().add(-1, 'day').toDate().toISOString().split("T")[0],
         names: [
             "yesterday"
         ]
     },
     {
-        value: moment().add(1, 'day').toDate(),
+        value: moment().add(1, 'day').toDate().toISOString().split("T")[0],
         names: [
             "tomorrow"
         ]
     },
 ];
 
-module.exports = {MealTimes, Dates};
+module.exports = {mealTimes, datesLog};
```

Comment:
- It's important to be consistent
```suggestion
module.exports = { mealTimes, datesLog };
```